src/services/api/index.ts

```diff
@@ -27,6 +26,7 @@ interface YdbEmbeddedAPIProps {
     proxyMeta: undefined | boolean;
     // this setting allows to use schema object path relative to database in api requests
     useRelativePath: undefined | boolean;
+    useMetaSettings: undefined | boolean;
```

Comment:
Moved `useMetaSettings` from `uiFactory` to api setup. The reason is that `uiFactory` property requires api and `uiFactory` to be inited in the specific order: first we need to `configureUIFactory` and only after setup api. Otherwise we need two settings - for `uiFactory` and to api.

So this help to reduce it to only one setting without any additional requirements